### ISSUE TITLE
Line number not working

### ISSUE THREAD

**Issue by nicolas-besnard** [rspec/rspec-rails#1385]
To run my test, I usually use: 

`bin/rspec spec/controllers/api/v1/votes_controller_spec.rb:23` 

but the line specification is no longer working. Is there any other issue with this ?

I was using version 3.2.0, it's working with 3.1.0

**Comment by myronmarston** [comment]
A regression in line-number filtering was reported in rspec/rspec-core#1961 and fixed in rspec/rspec-core#1963.  Does that describe your issue?  We plan to release 3.3 with the fix very soon.

If that doesn't describe your issue, you'll need to tell us more about your project (or at least the contents of that file)...or put together a repro case, because in general, line number filtering does work. I use it every day.  You may have hit some edge case specific to how your project is organized, though.

**Comment by nicolas-besnard** [comment]
This is exactly my issue :)

**Comment by myronmarston** [comment]
If you want a fix before rspec 3.3 is out, you can use HEAD by adding this to your `Gemfile`:

``` ruby
%w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
  gem lib, :git => "git://github.com/rspec/#{lib}.git", :branch => 'master'
end
```
